components/cldr-json-data-provider/src/error.rs

```diff
@@ -1,12 +1,16 @@
 use std::error;
 use std::fmt;
 
+#[cfg(feature = "download")]
+use crate::download::error::Error as DownloadError;
+
 #[non_exhaustive]
 #[derive(Debug)]
 pub enum Error {
     JsonError(serde_json::error::Error),
     IoError(std::io::Error, std::path::PathBuf),
     MissingSource(MissingSourceError),
+    Download(Box<dyn error::Error>),
```

Comment:
is there a reason you don't want to use `DownloadError` here?

Reply:
It was either use a box or make this `Download(DownloadError)` and protect that enum variant with `#[cfg(feature = "download")]` everywhere it is used in this file.  I had a slight preference for a box in order to reduce the number of places where I needed `#[cfg(feature = "download")]`.  Do you have a preference?

Reply:
OK, I made this change.